mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -292,19 +293,34 @@ public MessageManager getMailbox(MailboxId mailboxId, MailboxSession session)
         MailboxMapper mapper = mailboxSessionMapperFactory.getMailboxMapper(session);
         Mailbox mailboxRow = mapper.findMailboxById(mailboxId);
 
-        if (mailboxRow == null) {
```

Comment:
I can't find where you check for null now. Why did you removed it?

Reply:
No mapper returns null (jcr behaviour)

Reply:
interesting, however it deserves a separated commit, It can't be lost in the middle of this refactoring